tests/in/pointers.wgsl

```diff
@@ -0,0 +1,5 @@
+fn f() {
+   var v: vec2<i32>;
```

Comment:
can we just roll this into `access.wgsl` instead?

Reply:
I tried that at first. But we generate all the output formats for `access.wgsl`, and we don't have a pointer story yet for GLSL, HLSL, or even MSL. So I put this in its own file so that we could just generate WGSL and SPIR-V.

Reply:
ok, fair